### ISSUE TITLE
InflateException: Binary XML file line #110

### ISSUE THREAD

**Issue by nathan130200** [google/google-authenticator-android#119]
- [X] This issue is present in the app downloaded from the [Play Store][playstore].

## Describe the Bug

After last google authenticator update, i cannot open google authenticator app.

### Intended Behavior

Open google authenticator activity, displaying all my 2fa codes.

### Actual Behavior

Activity crashing at startup.

## Reproducing

1. Open app
2. See the app crashing.

### Device

Fill out the details about the device and app version in which you encountered the bug.

 - Device: Pocket23gxx (Samsung/SM G11-0B)
 - OS: Android 4.4.2
 - Version Google Authenticator 5.10

## Additional Information

Full exception stack trace (taken with logcat):

```java
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.google.android.apps.authenticator2/com.google.android.apps.authenticator.AuthenticatorActivity}: android.view.InflateException: Binary XML file line #110: Error inflating class com.google.android.libraries.material.speeddial.expandable.ExpandableFloatingActionButton
E/AndroidRuntime(17596):        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2436)
E/AndroidRuntime(17596):        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2498)
E/AndroidRuntime(17596):        at android.app.ActivityThread.access$900(ActivityThread.java:179)
E/AndroidRuntime(17596):        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1324)
E/AndroidRuntime(17596):        at android.os.Handler.dispatchMessage(Handler.java:102)
E/AndroidRuntime(17596):        at android.os.Looper.loop(Looper.java:146)
E/AndroidRuntime(17596):        at android.app.ActivityThread.main(ActivityThread.java:5641)
E/AndroidRuntime(17596):        at java.lang.reflect.Method.invokeNative(Native Method)
E/AndroidRuntime(17596):        at java.lang.reflect.Method.invoke(Method.java:515)
E/AndroidRuntime(17596):        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1288)
E/AndroidRuntime(17596):        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1104)
E/AndroidRuntime(17596):        at dalvik.system.NativeStart.main(Native Method)
E/AndroidRuntime(17596): Caused by: android.view.InflateException: Binary XML file line #110: Error inflating class com.google.android.libraries.material.speeddial.expandable.ExpandableFloatingActionButton
E/AndroidRuntime(17596):        at android.view.LayoutInflater.createView(LayoutInflater.java:626)
E/AndroidRuntime(17596):        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:702)
E/AndroidRuntime(17596):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:761)
E/AndroidRuntime(17596):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:769)
E/AndroidRuntime(17596):        at android.view.LayoutInflater.rInflate(LayoutInflater.java:769)
E/AndroidRuntime(17596):        at android.view.LayoutInflater.inflate(LayoutInflater.java:498)
E/AndroidRuntime(17596):        at android.view.LayoutInflater.inflate(LayoutInflater.java:398)
E/AndroidRuntime(17596):        at android.view.LayoutInflater.inflate(LayoutInflater.java:354)
E/AndroidRuntime(17596):        at android.support.v7.app.AppCompatDelegateImpl.setContentView(PG:819)
E/AndroidRuntime(17596):        at android.support.v7.app.AppCompatActivity.setContentView(PG:117)
E/AndroidRuntime(17596):        at com.google.android.apps.authenticator.AuthenticatorActivity.onCreate(PG:234)
E/AndroidRuntime(17596):        at android.app.Activity.performCreate(Activity.java:5484)
E/AndroidRuntime(17596):        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1093)
E/AndroidRuntime(17596):        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2400)
```

[playstore]: https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2
[opensource]: https://github.com/google/google-authenticator-android

**Comment by nathan130200** [comment]
@google-admin @google Hello???????

**Comment by nathan130200** [comment]
Closing issue due inactivity. Also i fixed with #70 reverting Authenticator app version to previous version (also my codes works too).
